superset/db_engine_specs.py

```diff
@@ -37,6 +42,7 @@ class LimitMethod(object):
 
 class BaseEngineSpec(object):
     engine = 'base'  # str as defined in sqlalchemy.engine.engine
+    async = False
```

Comment:
Another more generic option here would be to call this `cursor_execute_kwargs` and make it:
```
cursor_execute_kwargs = {'async': True}
```
for Presto and Hive